components/combobox/stories/combobox.stories.js

```diff
@@ -64,151 +90,66 @@ export default {
 		isDisabled: false,
 		showFieldLabel: false,
 		testId: "combobox",
+		content: [
+			(passthroughs, context) => Menu({
+				role: "listbox",
+				subrole: "option",
+				selectionMode: "single",
+				hasDividers: true,
+				items: [
+					{
+						label: "Ballard",
+						isSelected: true,
+						isChecked: true,
+					},
+					{
+						label: "Fremont",
+					},
+					{
+						label: "Greenwood",
+					},
+					{
+						type: "divider",
+					},
+					{
+						label: "United States of America",
+						isDisabled: true,
+					},
+				],
+				...passthroughs,
+			}, context),
+		],
 	},
 	parameters: {
 		packageJson: pkgJson,
 	},
 };
 
 export const Default = ComboBoxGroup.bind({});
+Default.tags = ["!autodocs"];
 Default.args = {
 	isOpen: true,
 	fieldLabelText: "Select location",
-	content: [
-		Menu({
-			role: "listbox",
-			subrole: "option",
-			isSelectable: true,
-			items: [
-				{
-					label: "Ballard",
-					isSelected: true,
-					isChecked: true,
-				},
-				{
-					label: "Fremont",
-				},
-				{
-					label: "Greenwood",
-				},
-				{
-					type: "divider",
-				},
-				{
-					label: "United States of America",
-					isDisabled: true,
-				},
-			],
-		}),
-	],
+	value: "Ballard",
```

Comment:
Does this need to be in the controls for any reason? Even if we hid it from the controls table? I don't know, I was just curious.

Reply:
Yeah, we have it for Picker, I don't see a reason not to add one for Combobox!